client/js/ifml2code/client/elementrules/templates/system-event-vm.js.ejs

```diff
@@ -0,0 +1,51 @@
+/*jslint node: true, nomen: true */
+"use strict";
+
+var ko = require('knockout');
+
+function ViewModel(params) {
+    var self = this;
+
+    // TODO: Execution
+    /*
+      example:
+        //to trigger the event: params.fire;
+        document.addEventListener('keydown', params.fire);
```

Comment:
It is probably better to listen with the `trigger` function, in order to make the example clearer